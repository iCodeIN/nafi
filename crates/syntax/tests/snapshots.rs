use nafi_syntax::SourceFile;

#[test]
fn parse_snapshot_tests() {
    insta::glob!("snapshots/*.nafi", |path| {
        let input = std::fs::read_to_string(path).unwrap();
        insta::assert_debug_snapshot!(SourceFile::parse(&input).syntax_tree());
    });
}
