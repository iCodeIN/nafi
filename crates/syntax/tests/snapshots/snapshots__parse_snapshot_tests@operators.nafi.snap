---
source: crates/syntax/tests/snapshots.rs
expression: "SourceFile::parse(&input).syntax_tree()"
---
SourceFile@0..9
  Expr@0..9
    Expr@0..7
      Expr@0..1
        LitDigits@0..1 "1"
      Syntax@1..2 "+"
      Expr@2..7
        Expr@2..5
          Expr@2..3
            LitDigits@2..3 "2"
          Syntax@3..4 "*"
          Expr@4..5
            LitDigits@4..5 "3"
        Syntax@5..6 "/"
        Expr@6..7
          LitDigits@6..7 "4"
    Syntax@7..8 "-"
    Expr@8..9
      LitDigits@8..9 "5"

